teamcity/pyproject.toml

```
@@ -61,3 +59,7 @@ include = [
 dev-mode-dirs = [
     ".",
 ]
+
+[tool.pytest.ini_options]
+norecursedirs = "tests/docker"
```

Maybe it needs a leading slash? The way you have it now runs for me locally on py38

```suggestion
norecursedirs = "/tests/docker"
```